.buildkite/steps/release-homebrew.sh

```diff
@@ -71,7 +72,7 @@ echo "Release SHA256: $RELEASE_SHA256"
 
 echo "--- :octocat: Fetching current homebrew formula from Github Contents API"
 
-CONTENTS_API_RESPONSE="$(curl "https://api.github.com/repos/buildkite/homebrew-buildkite/contents/buildkite-agent.rb?access_token=$GITHUB_RELEASE_ACCESS_TOKEN")"
```

Comment:
updating the way we authenticate to the github API, as recommended by github: https://developer.github.com/changes/2020-02-10-deprecating-auth-through-query-param/